packages/e2e/cypress/integration/ui/components/authenticator/sign-in-totp-mfa/sign-in-totp-mfa.steps.ts

```diff
@@ -1,4 +1,7 @@
 import { Then, When } from '@badeball/cypress-cucumber-preprocessor';
+import { Decoder } from '@nuintun/qrcode';
```

Comment:
This library is a dev dependency used to get the QR code information directly from the image